app/controllers/admin/products_controller.rb

```diff
@@ -44,8 +58,8 @@ def destroy
 
   private
   def product_params
-    params.require(:product).permit(:brand_id, :category_id, :name, :description, 
-      :price, :quantity, :image).tap do |while_listed|
+    params.require(:product).permit(:brand_id, :category_id, :name, :description,
+      :price, :quantity, pictures_attributes: [:id, :file, :_destroy, :remove_file]).tap do |while_listed|
```

Comment:
Line is too long. [106/80]<br>Align the parameters of a method call if they span more than one line.